### PR TITLE
Fixed #339

### LINKED ISSUE #339: "early_stopping_metric" doesn't seem to work
I am trying multi_label and multi_class classification with early stopping.

I have insert the sklearn.metrics.f1(average = "micro) function into **kwargs with using this code:

               ` kwargs = {"f1_micro": partial(sklearn.metrics.f1_score, average = "micro")}`

The f1_micro score is being calculated on the eval_df and it gets printed while training. But the argument :

"early_stopping_metric" = "f1_micro" doesn't work. It seems that it always calculates the loss function on eval_df for the early stopping functionality.
It is my bad or there is indeed a bug?

### PR DESCRIPTION
**Pull request by djstrong**

### CODE REVIEW

File: simpletransformers/classification/classification_model.py [hunk 1]
```diff
@@ -501,7 +501,7 @@ def train(
                             best_eval_metric = results[args.early_stopping_metric]
                             self._save_model(args.best_model_dir, optimizer, scheduler, model=model, results=results)
                         if best_eval_metric and args.early_stopping_metric_minimize:
-                            if results[args.early_stopping_metric] - best_eval_metric < args.early_stopping_delta:
+                            if best_eval_metric - results[args.early_stopping_metric] > args.early_stopping_delta:
```
Comment (ThilinaRajapakse) [inline simpletransformers/classification/classification_model.py:504]: Aren't both these conditions the same? :thinking:

Comment (djstrong) [reply simpletransformers/classification/classification_model.py:504]: The change is the same as: `if results[args.early_stopping_metric] - best_eval_metric < - args.early_stopping_delta:`

Comment (ThilinaRajapakse) [reply simpletransformers/classification/classification_model.py:504]: Got it, my bad.
